packages/react-aria-components/src/GridList.tsx

```diff
@@ -561,3 +562,59 @@ export const GridListLoadMoreItem = createLeafComponent('loader', function GridL
     </>
   );
 });
+
+export interface GridListSectionProps<T> extends SectionProps<T> {}
+
+/**
+ * A GridListSection represents a section within a GridList.
+ */
+export const GridListSection = /*#__PURE__*/ createBranchComponent('section', <T extends object>(props: GridListSectionProps<T>, ref: ForwardedRef<HTMLElement>, item: Node<T>) => {
+  let state = useContext(ListStateContext)!;
+  let {dragAndDropHooks, dropState} = useContext(DragAndDropContext)!;
```

Comment:
Can you drop on a section? or drag one? do we have any stories covering this? or tests?

Reply:
yeah i think we can remove it for now